Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml

```diff
@@ -553,6 +553,34 @@ script:
       required: true
       description: The name of the tags to apply to supplied assets.
     description: Assigns tags to a list of assets.
+  - name: asm-add-note-to-asset
+    arguments:
+    - name: asset_id
+      description: The asset ID to add a note to.
+      required: true
+    - name: entity_type
+      description: "The type of Xpanse asset. Allowed values are: 'asset' or 'ip_range'."
```

Comment:
```suggestion
      description: "The type of Cortex Xpanse asset. 
```

Reply:
Only those 2 types are allowed. I don't see why we would remove them from the description the user sees.